package/lib/SimpleSchema_allowedValues.tests.js

```diff
@@ -303,5 +375,22 @@ describe('SimpleSchema - allowedValues', function () {
       });
       expect(schema.getAllowedValuesForKey('foo')).toEqual(allowedValues);
     });
+
+    it('works with set, convert to array', function () {
+      if (typeof Set === 'function') {
```

Comment:
It would be better to not have this `if` because that might cause it to appear as if the test passes when really `Set` is just undefined. If the current babel config for Jest runs doesn't already polyfill this, you could set it up to do so.